DOCS-7870-made edits (#63)
mulesoft/docs-service-mesh: modules/ROOT/pages/prepare-to-install-service-mesh.adoc

. Save the `.bashrc` file.
. Restart your terminal.

> == Preparing to Install Instio

Reviewer: ```suggestion
== Preparing to Install Istio
```

Reviewer 2: Be sure to fix this GLOBALLY. It looks like your reviewer has caught them all, but double-check after you update the PR.
Also, please check for and resolve reviewer comments BEFORE submitting PRs for editing. Thanks!

Reviewer 2: After reading further, I see that this topic is NOT about PREPARING to install; it is about installing different versions of Instio. Please clarify.

Author: Done! Thanks!

> When you install Instio in your environment, you must configure Instio settings based on the Instio version that you install.

Reviewer: ```suggestion
When you install Istio in your environment, you must configure Istio settings based on the Istio version that you install.
```

Reviewer 2: I don't understand the purpose of this sentence; it seems to be stating the obvious, and in a bit of a confusing way.
If you install a particular version, don't the settings come with that version?
Are you trying to say that the audience needs to prepare its environment for the particular version of Istio it is installing? Or that different versions require different prep work?

Reviewer 2: I now see that this is not about preparing but about how different Istio versions are installed in slightly different ways. Clarify this for the audience.

Author: Done! Thanks!

> === Installing Instio 1.4x

Reviewer: ```suggestion
=== Installing Istio 1.4.x
```

Reviewer 2: GLOBALLY search for this version typo as well.

Author: Done! Thanks!

> To install and configure Instio 1.4x for Anypoint Service Mesh:

Reviewer: ```suggestion
To install and configure Istio 1.4.x for Anypoint Service Mesh:
```

Author: Done! Thanks!

> . Download and install Instio using the Instio documentation.

Reviewer: ```suggestion
. Download and install Istio using the Istio documentation.
```

Reviewer 2: This topic is titled PREPARING to install, not installing. I think you need to retitle the container topic....

Author: Done! Thanks!

. Enable the policy control flag:
+

> `set values.global.disablePolicyChecks=false`

Reviewer: ```suggestion
`--set values.global.disablePolicyChecks=false`
```

Author: Done! Thanks!

+

> For more information about this flag, see https://archive.istio.io/v1.4/docs/tasks/policy-enforcement/enabling-policy/[Policy Enforcement]. 

Reviewer 2: Would your audience really need to look this up to perform this task? 
I think this is probably unnecessary.

Author: I'd leave it in for people who need more clarity about what the flag does. Done! Thanks!

> === Installing Instio 1.5x

Reviewer: ```suggestion
=== Installing Istio 1.5.x
```

Author: Done! Thanks!

> To install and configure Instio 1.5x for Anypoint Service Mesh:

Reviewer: ```suggestion
To install and configure Istio 1.5.x for Anypoint Service Mesh:
```

Author: Done! Thanks!

> . Download and install Instio using the Instio documentation.

Reviewer: ```suggestion
. Download and install Istio using the Istio documentation.
```

Author: Done! Thanks!

. Enable the policy control flag:
+
[source,text,linenums]
----
values:
  global:
    disablePolicyChecks: false
  components:
    policy:
      enabled: true
    sidecarInjector:
      enabled: true
----
. Enable telemetry:
+
[source,text,linenums]
----
values:
  prometheus:
    enabled: true
  telemetry:
    v1:
      enabled: true
    v2:
      enabled: false
components:
  citadel:
    enabled: true
  telemetry:
    enabled: true
----

> If you configured your profile correctly with these configurations settings, a full manifest file is displayed as illustrated in the following example:

Reviewer 3: Hi, 
You changed the meaning of what I had placed. "An example of the full manifest file would be. If using it, remember to choose the correct profile."

See that there is a field that is called profile.

Author: Done! Thanks!

> If you configured your profile correctly with these configurations settings, a full manifest file is displayed as illustrated in the following example:

Reviewer 2: This sentence needs to be changed per reviewer comment.

Author: Done! Thanks!

+

> ----
> [source,text,linenums]

Reviewer: ```suggestion
[source,text,linenums]
----
```

Author: Done! Thanks!

apiVersion: install.istio.io/v1alpha1
kind: IstioOperator

> spec:

Reviewer 3: ```suggestion
spec:
  profile: default
  components:
    policy:
      enabled: true
    sidecarInjector:
      enabled: true
    citadel:
      enabled: true
    telemetry:
      enabled: true
  addonComponents:
    prometheus:
      enabled: false
  values:
    global:
      disablePolicyChecks: false
    telemetry:
      v1:
        enabled: true
      v2:
        enabled: false
```

It was missing the actual spec 🤔

Reviewer 2: Please update per reviewer comment. 
I really can't edit something that continues to change.

Author: @tbruno the actual file you sent me was missing his portion. :-)

Reviewer 3: mmm, it is here: https://docs.google.com/document/d/1pwcuRVylXUvfj2LqkbYrEBk2j2YQKUv7UwwCWqjzo-o/edit
